docs/tutorials/repository-tutorial/01-modeling-and-features.rst

```diff
@@ -0,0 +1,41 @@
+Introduction to Database Modeling and Repository Features
+---------------------------------------------------------
+In this tutorial, we will cover the integrated repository features in Litestar 2.0, starting with database modeling using the included SQLAlchemy declarative model helpers. These are a series of classes and mixins that incorporate commonly used functions/column types to make working with models easier.
+
+.. literalinclude:: /examples/contrib/sqlalchemy/sqlalchemy_declarative_models.py
+    :language: python
+    :caption: app.py
+    :emphasize-lines: 9, 8,19,20
+    :linenos:
+
+We'll begin by modelling the entities and relationships between authors and books.  We'll start by creating the ``Author`` table, utilizing the :class:`UUIDBase <litestar.contrib.sqlalchemy.base.UUIDBase>` class. To keep things simple, our first model will encompass only three fields: `id`, `name`, and `dob`.
+
+The books entity is not considered a "strong" entity and therefore always requires an author to be created.  We need to configure our SQLAlchemy classes so that it is aware of this relationship.
+We will extend the ``Author`` model by incorporating a ``Book`` relationship. This would allow each ``Author`` record to possess multiple ``Book`` records. By configuring it this way, SQLAlchemy will automatically include the necessary foreign key constraints when using the `author_id` field in each ``Book`` record.
```

Comment:
```suggestion
We will extend the ``Author`` model by incorporating a ``Book`` relationship. This would allow each ``Author`` record to possess multiple ``Book`` records. By configuring it this way, SQLAlchemy will automatically include the necessary foreign key constraints when using the ``author_id`` field in each ``Book`` record.
```